src/System.Text.Primitives/System/Text/Formatting/PrimitiveFormatter.Time.cs

```diff
@@ -19,15 +19,16 @@ public static bool TryFormat(this DateTimeOffset value, Span<byte> buffer, out i
                 offset = value.Offset;
             }
 
-            Precondition.Require(format.Symbol == 'R' || format.Symbol == 'O' || format.Symbol == 'G');
-
             symbolTable = symbolTable ?? SymbolTable.InvariantUtf8;
 
             switch (format.Symbol)
             {
                 case 'R':
                     return TryFormatDateTimeRfc1123(value.UtcDateTime, buffer, out bytesWritten, symbolTable);
 
+                case 'r':
```

Comment:
Should this instead just be as follows (without needing the new method TryFormatDateTimeRfc1123Lowercase):

```C#
case 'R':
case 'r':
    return TryFormatDateTimeRfc1123(value.UtcDateTime, buffer, out bytesWritten, symbolTable); 
```

If not, can you clarify my misunderstanding?